pycaret/internal/helper.py

```diff
@@ -0,0 +1,677 @@
+# report helper
```

Comment:
Please move this to a separate folder (eg. pycaret/internal/report_generator/helper.py)

Reply:
DONE